test/minigzip.c

```diff
@@ -33,8 +33,12 @@
 #endif
 
 #ifndef UNALIGNED_OK
+#ifdef __FreeBSD__
+#  include <stdlib.h>
+#else
 #  include <malloc.h>
```

Comment:
Maybe use `stdlib.h` unconditionally?

https://stackoverflow.com/questions/12973311/difference-between-stdlib-h-and-malloc-h

Reply:
@myfreeweb Current version of this file doesn't use memalign() for allocating buffers, but it might be faster if it would use aligned allocation instead of malloc().

Reply:
btw, there's the standardized `posix_memalign` that should be used for this purpose..

Reply:
posix_memalign() is not always available... Even if something is standardized, it doesn't mean all compilers honor that standard...